lib/ws.js

```diff
@@ -69,6 +72,35 @@ class BinanceWS {
         return data;
     }
 
+    _clearUserDataInterval() {
+        this.userDataRefreshIntervalId &&
+            clearInterval(this.userDataRefreshIntervalId);
+        this.userDataRefreshIntervalId = false;
+    }
+
+    _sendUserDataKeepAlive(binanceRest, response) {
+        return binanceRest.keepAliveUserDataStream(response).catch(e => {
+            this.userDataRefreshIntervalId.failCount++;
+            const msg =
+                'Failed requesting keepAliveUserDataStream for onUserData listener';
+            if (e && e.code === -1125) {
```

Comment:
Could we perhaps define -1125 as a more descriptive constant and use that instead?  Something like `const INVALID_LISTEN_KEY = -1125` at the top of the file.

Reply:
How's this? I'm hesitant to populate a full list of errors (and then maintain it), but this at least leaves a placeholder for it, while addressing this request.

Reply:
No need for a full list.  My usual preference is just avoid [magic numbers](https://en.wikipedia.org/wiki/Magic_number_(programming)#Unnamed_numerical_constants).  We can add constants as needed in the future